tabortbok

TomasSteifo opened the pull request:
har lagt till ta bort funktion

TomasSteifo committed:
tabortbok

OscarJerez reviewed (changes requested):
snyggt men ändra på meny val då det har kommit nya menyval samt lägga in de antal meny siffror som behövs

RichardErenmalm committed:
ändrade i program att 4 är avsluta istället för 3

Jag tror det var det som var problemet, det fungerar på min dator nu iallafall. Någon annan kan testa och accepta pullrequesten så vi vet att det inte bara fungerar för mig

OscarJerez reviewed (approved):
såg att Richard gjorde ändring

RichardErenmalm committed:
Merge branch 'master' into Tomasandringar

RichardErenmalm committed:
Update UserInteraktion.cs

RichardErenmalm committed:
Update Program.cs

tog bort accidental dubbelkod

RichardErenmalm reviewed (changes requested):
Det verkar fungera för mig men hade varit bra om ni kollade också om jag missat något
